### DISCUSSION TITLE
Philips Hue tap dials all have the same ieeeAddr

### DISCUSSION THREAD

**Discussion by savdagod** [Koenkk/zigbee2mqtt#16698] [Devices]
I’ve bought 5 taps so far that all have the same ieeeAddr. Anyone have the same issue? Because of this, you can only have one on your network at a time.

**Comment by lux73** [comment]
this should not be possible - if you could confirm they are not cheap replica's i would contact the dealer they are from and return

afaik its not possible to change the ieeeAddr on non Coordinator Devices

**Comment by savdagod** [comment]
Incase anyone comes across this, on brand new remotes you hold button 1 for 3 sec and it will instantly pair. I edited the Z2M doc for this device to state this (it says to hold reset, which only works AFTER you’ve use button 1 to pair it). Turns out, the remote I would unpair (to test my theory in the OP) would automatically re-pair. I was thinking the new remote was pairing, when in fact it wasn’t doing anything. After 6 remotes, I found this out by just reading the manual.
